### ISSUE TITLE
Question regarding BaslineEmbeddingNet.

### ISSUE THREAD

**Issue by Axe--** [rafellerc/Pytorch-SiamFC#14]
As per the [SiamFC paper](https://arxiv.org/pdf/1606.09549.pdf), the embedding module architecture (Table 1) had _stride=2_ for pool_2.
In models.py, [line 34](https://github.com/rafellerc/Pytorch-SiamFC/blob/master/training/models.py#L34)
shouldn't stride = 2? 

`self.fully_conv = nn.Sequential(nn.Conv2d(3, 96, kernel_size=11,
                                                  stride=2, bias=True),
                                        nn.BatchNorm2d(96),
                                        nn.ReLU(),
                                        nn.MaxPool2d(3, stride=2),

                                        nn.Conv2d(96, 256, kernel_size=5,
                                                  stride=1, groups=2,
                                                  bias=True),
                                        nn.BatchNorm2d(256),
                                        nn.ReLU(),
                                        nn.MaxPool2d(3, stride=1),    # <<<<<------------
                                        nn.Conv2d(256, 384, kernel_size=3,
                                                  stride=1, groups=1,
                                                  bias=True),
                                        nn.BatchNorm2d(384),
                                        nn.ReLU(),
                                        nn.Conv2d(384, 384, kernel_size=3,
                                                  stride=1, groups=2,
                                                  bias=True),
                                        nn.BatchNorm2d(384),
                                        nn.ReLU(),
                                        nn.Conv2d(384, 32, kernel_size=3,
                                                  stride=1, groups=2,
                                                  bias=True))`

**Comment by rafellerc** [comment]
Yes, but the actual SiamFC version that won the VOT-17 challenge is an updated version that is briefly presented in their paper [Learning feed-forward one-shot learners](https://arxiv.org/abs/1606.05233), which corresponds to their released codes (the Matlab and TensoFlow versions). You can check that in https://github.com/torrvision/siamfc-tf/blob/59cbf8384b925a0f2097a2b2f9a15752fc774b01/src/siamese.py#L22
